pkg/util/version/clusterversion.go

```diff
@@ -26,3 +26,12 @@ func GetClusterVersion(ctx context.Context, configcli configclient.Interface) (*
 
 	return nil, errors.New("unknown cluster version")
 }
+
+func GetClusterDesiredVersion(ctx context.Context, configcli configclient.Interface) (*Version, error) {
+	cv, err := configcli.ConfigV1().ClusterVersions().Get(ctx, "version", metav1.GetOptions{})
```

Comment:
I'm not convinced that this deserves own exported function.

Reply:
I suspect we will need this in the future too :/ patterns matching on clusterVersion does not work quite well.